scripts/smoke-test-eslint-version-conflicts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
I have too many comments on this file (because I have been traumatized by `shellcheck`). Here is a diff to apply:

```diff
diff --git a/scripts/smoke-test-eslint-version-conflicts b/scripts/smoke-test-eslint-version-conflicts
index 5f2ec1b1..2bb78afb 100755
--- a/scripts/smoke-test-eslint-version-conflicts
+++ b/scripts/smoke-test-eslint-version-conflicts
@@ -2,14 +2,14 @@
 
 set -e
 
-LINTER_PATH=`pwd`
-TARGET_TEST_PATH=$LINTER_PATH/tmp/test-project
-FIXTURE_TEST_PATH=$LINTER_PATH/tests/fixtures/eslint-versions-conflicts-package
-LINTER_VERSION=`node -p 'require("./package.json").version'`
+LINTER_PATH=$(pwd)
+TARGET_TEST_PATH="$LINTER_PATH/tmp/test-project"
+FIXTURE_TEST_PATH="$LINTER_PATH/tests/fixtures/eslint-versions-conflicts-package"
+LINTER_VERSION=$(node -p 'require("./package.json").version')
 
 # Fail if the target path already exists.
 [[ -f "$TARGET_TEST_PATH" ]] && (
-  echo "ERROR: Temporary test dir '$TARGET_TEST_PATH' already exist."
+  echo "ERROR: Temporary test dir '$TARGET_TEST_PATH' already exists."
   exit 1
 )
 
@@ -20,15 +20,15 @@ npm ci
 npm run build
 
 # Create target test path directory and cleanup on exit.
-cp -rf $FIXTURE_TEST_PATH $TARGET_TEST_PATH
-trap "rm -rf $TARGET_TEST_PATH" EXIT
+cp -rf "$FIXTURE_TEST_PATH" "$TARGET_TEST_PATH"
+trap 'rm -rf "$TARGET_TEST_PATH"' EXIT
 
 pushd "$TARGET_TEST_PATH"
-npm pack $LINTER_PATH
+npm pack "$LINTER_PATH"
 npm install
-npm install --save-dev ./addons-linter-$LINTER_VERSION.tgz
+npm install --save-dev "./addons-linter-$LINTER_VERSION.tgz"
 
-echo -e "INFO: Run addons-linter on the test fixture project\n" 
+echo -e "INFO: Run addons-linter on the test fixture project\n"
 
 npx addons-linter ./src || (
   echo "ERROR: addons-linter validation failed".
```

Reply:
@willdurand ah, thanks! Actually I should have thought of running shellcheck on the script. Diff applied in the last updated push on this pull request (I rolled it into the second patch patch of this pull request)